### PR TITLE
Admin: Remove new and beta badges

### PR DESCRIPTION
**Pull request by zhen0**
## Description
<! -- What is it meant to do? -->
Removes "new" and "beta" badges

## Linked Issues
<! -- Use a key word (e.g. closes or resolves) to close related issues  -->

## Tests and performance

 - [x] Changes to any .js files are covered by existing tests or this PR adds new tests (if not please explain why below)
 - [x] No new packages are added or package size and performance considerations are discussed below
 - [x] PR Title fits our [changelog format](https://github.com/PrefectHQ/ui#readme)

 ## Releases/Changelog cuts only
 - [ ] Completed the [QA Checklist](https://www.notion.so/prefect/Cloud-UI-QA-Checklist-038a5a5d40a249d78232917ad1b923b9) in staging

### CODE REVIEW

**Review by stackoverfloweth — DISMISSED** [review]

File: src/components/Nav/TeamMenu.vue [hunk 1]
```diff
@@ -230,6 +202,32 @@ export default {
...
+          </v-list-item-avatar>
+          <v-list-item-content>
+            <v-list-item-title class="text-subtitle-1">
+              API Tokens
+            </v-list-item-title>
+            <v-list-item-subtitle>
+              Manage your team's API tokens
+            </v-list-item-subtitle>
+          </v-list-item-content>
+        </v-list-item>
```
Comment (znicholasbrown) [inline src/components/Nav/TeamMenu.vue:230]: Can we move these back up the list and alphabetize the rest of the list? These were meant to be ordered but that got lost at some point

File: src/pages/FlowRun/FlowRun.vue [hunk 2]
```diff
@@ -77,7 +77,6 @@ export default {
           name: 'Artifacts',
           target: 'artifacts',
           icon: 'fas fa-fingerprint',
-          badgeText: 'Beta',
           cardText:
             'The Artifacts API is a beta feature currently under development. Task mapping with artifacts may have unexpected results... for more information on artifacts, check out the',
           cardLink: 'https://docs.prefect.io/api/latest/backend/artifacts.html',
```
Comment (znicholasbrown) [inline src/pages/FlowRun/FlowRun.vue:82]: Can you remove this card text and link as well?

File: src/pages/TaskRun/TaskRun.vue [hunk 3]
```diff
@@ -86,15 +86,12 @@ export default {
...
           hidden: !this.mappedParent && !this.mappedChild
         },
         {
           name: 'Artifacts',
           target: 'artifacts',
           icon: 'fas fa-fingerprint',
-          badgeText: 'Beta',
           cardText:
             'The Artifacts API is a beta feature currently under development. Task mapping with artifacts may have unexpected results... for more information on artifacts, check out the',
           cardLink: 'https://docs.prefect.io/api/latest/backend/artifacts.html',
```
Comment (znicholasbrown) [inline src/pages/TaskRun/TaskRun.vue:97]: Can you remove this card text and link as well?

**Review by znicholasbrown — CHANGES_REQUESTED** [review]
A few comments @zhen0

File: src/components/Nav/TeamMenu.vue (hunk 1 above)
Comment (zhen0) [reply src/components/Nav/TeamMenu.vue:230]: I moved these as I dislike having the deprecated API Tokens at the top of the list.  However we discussed and agreed that for consistency we should keep the list alphabetized- I'll update.

**Comment by zhen0** [comment]
Github makes the alphabetization of the team menu look more complicated than it is is.  Here's what the re-rog looks like:

Screen Shot 2022-02-15 at 11 39 36 AM

**Review by znicholasbrown — APPROVED** [review]
Thanks @zhen0 !

**Pull request merged**